### PR TITLE
DEVDOCS-4704: [update] Global objects, create new section for urls

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4704]

## What changed?
Moved urls object into its own section. See dev doc ticket for details.

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4704]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4704?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by slsriehl** [comment]
@bc-traciporter please edit PR title

**Comment by bc-bm** [comment]
Hi Traci

We need to add a note around "payment_methods" that it will be visible when "[Stored payment methods](https://support.bigcommerce.com/s/article/Enabling-Stored-Payment-Methods?language=en_US)"  are enabled on the store. Else this field wouldn't be sent via the variable.

Thanks,
Bala

**Comment by bc-traciporter** [comment]
@bc-bm Thank you for letting me know. I added the text as per your request.

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 1]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+## Urls
+
+**Description:** Object containing a collection of storefront pages and their sub-pages<br>
+
+**Handlebars Expression:** `{{urls}}`
+
+**Object Properties:** 
+
+| Property | Description |
+|-|-|
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:576]:
```suggestion
| :--- | :--- |
```
I believe we've been doing this for our tables

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 2]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| urls | Global URLs that the template can access – for example, the template could link to the cart page using {{urls.cart}} |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:577]:
```suggestion
| urls | Global URLs that the template can access – for example, the template could link to the cart page using {{urls.cart}}. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 3]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; home | Store’s home page |
+| &#x21B3; account | Collection of (customer and storefront) account-related URLs:|
+| &nbsp; &nbsp; &#x21B3; index | Account index page `/account.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:580]:
```suggestion
| &nbsp; &nbsp; &#x21B3; index | Account index page `/account.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 4]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; orders | Collection of orders-related URLs: |
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; all | List of all orders. `/account.php?action=order_status` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:582]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; all | List of all orders. `/account.php?action=order_status`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 5]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; completed | URL to view completed orders `/account.php?action=view_orders` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:583]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; completed | URL to view completed orders `/account.php?action=view_orders`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 6]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+|  &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; save_new_return | URL to submit a New Return form<. `/account.php?action=save_new_return` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:584]:
```suggestion
|  &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; save_new_return | URL to submit a New Return form<. `/account.php?action=save_new_return`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 7]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; update_action | URL to submit an Edit Account form. `/account.php?action=update_account` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:585]:
```suggestion
| &nbsp; &nbsp; &#x21B3; update_action | URL to submit an Edit Account form. `/account.php?action=update_account`. |

```

Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:585]:
```suggestion
| &nbsp; &nbsp; &#x21B3; update_action | URL to submit an Edit Account form. For example, `/account.php?action=update_account`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 8]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; returns | List of returns. `/account.php?action=view_returns` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:586]:
```suggestion
| &nbsp; &nbsp; &#x21B3; returns | List of returns. For example, `/account.php?action=view_returns`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 9]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+|  &nbsp; &nbsp; &#x21B3; addresses | List of addresses; default sorting is by address id, from lowest to highest. `/account.php?action=address_book` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:587]:
```suggestion
|  &nbsp; &nbsp; &#x21B3; addresses | List of addresses; default sorting is by address id, from lowest to highest. For example, `/account.php?action=address_book`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 10]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; inbox | List of messages in customer’s inbox. `/account.php?action=inbox` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:588]:
```suggestion
| &nbsp; &nbsp; &#x21B3; inbox | List of messages in customer’s inbox. For example, `/account.php?action=inbox`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 11]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; send_message | URL to submit a message to the merchant. `/account.php?action=send_message` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:589]:
```suggestion
| &nbsp; &nbsp; &#x21B3; send_message | URL to submit a message to the merchant. For example, `/account.php?action=send_message`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 12]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; add_address | Link to add shipping address form. `/account.php?action=add_shipping_address` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:590]:
```suggestion
| &nbsp; &nbsp; &#x21B3; add_address | Link to add shipping address form. For example, `/account.php?action=add_shipping_address`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 13]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; wishlists | Collection of wishlist-related URLs: |
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; add | URL to the Add Wishlist form `/wishlist.php?action=addwishlist` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:592]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; add | URL to the Add Wishlist form. For example, `/wishlist.php?action=addwishlist`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 14]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; edit | URL to the Edit Wishlist form `/wishlist.php?action=editwishlist` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:593]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; edit | URL to the Edit Wishlist form. For example, `/wishlist.php?action=editwishlist`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 15]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; delete | URL to delete a wishlist `/wishlist.php?action=deletewishlist` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:594]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; delete | URL to delete a wishlist. For example, `/wishlist.php?action=deletewishlist`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 16]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; all | URL to view all wishlists `/wishlist.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:595]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; all | URL to view all wishlists. For example, `/wishlist.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 17]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; details | URL to the Edit Account form `/account.php?action=account_details` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:596]:
```suggestion
| &nbsp; &nbsp; &#x21B3; details | URL to the Edit Account form. For example, `/account.php?action=account_details`.  |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 18]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; recent_items | URL to a list of recently viewed items `/account.php?action=recent_items` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:597]:
```suggestion
| &nbsp; &nbsp; &#x21B3; recent_items | URL to a list of recently viewed items. For example, `/account.php?action=recent_items`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 19]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; payment_methods | Collection of payment methods available on a store, visible when [stored payment methods](https://support.bigcommerce.com/s/article/Enabling-Stored-Payment-Methods?language=en_US) are enabled: |
+| &nbsp; &nbsp; &#x21B3; all | URL to a form for payment methods `/account.php?action=payment_methods` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:599]:
```suggestion
| &nbsp; &nbsp; &#x21B3; all | URL to a form for payment methods. For example, `/account.php?action=payment_methods`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 20]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; brands | URL to view all brands; default sorting is by brand id, from lowest to highest |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:600]:
```suggestion
| &#x21B3; brands | URL to view all brands; default sorting is by brand ID, from lowest to highest. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 21]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; gift_certificate | Collection of gift-certificate–related URLs: |
+| &nbsp; &nbsp; &#x21B3; purchase | URL to a form for purchasing a gift certificate `/giftcertificates.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:602]:
```suggestion
| &nbsp; &nbsp; &#x21B3; purchase | URL to a form for purchasing a gift certificate. For example, `/giftcertificates.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 22]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; redeem | URL to view redemption information `/giftcertificates.php?action=redeem` |
+| &nbsp; &nbsp; &#x21B3; balance | URL to check the balance of a gift certificate `/giftcertificates.php?action=balance` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:604]:
```suggestion
| &nbsp; &nbsp; &#x21B3; balance | URL to check the balance of a gift certificate. For example, `/giftcertificates.php?action=balance`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 23]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; auth | Collection of authorization-related URLs: |
+| &nbsp; &nbsp; &#x21B3; login | URL to the login form `/login.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:606]:
```suggestion
| &nbsp; &nbsp; &#x21B3; login | URL to the login form. For example, `/login.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 24]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; check_login | URL to which to submit the login form `/login.php?action=check_login` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:607]:
```suggestion
| &nbsp; &nbsp; &#x21B3; check_login | URL to which to submit the login form. For example, `/login.php?action=check_login`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 25]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; create_account | URL to the Create Account form page `/login.php?action=create_account` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:608]:
```suggestion
| &nbsp; &nbsp; &#x21B3; create_account | URL to the Create Account form page. For example, `/login.php?action=create_account`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 26]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; save_new_account | URL to which to submit the Create Account form< `/login.php?action=save_new_account` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:609]:
```suggestion
| &nbsp; &nbsp; &#x21B3; save_new_account | URL to which to submit the Create Account form. For example, `/login.php?action=save_new_account`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 27]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; forgot_password | URL to the Forgot Password form `/login.php?action=reset_password` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:610]:
```suggestion
| &nbsp; &nbsp; &#x21B3; forgot_password | URL to the Forgot Password form. For example, `/login.php?action=reset_password`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 28]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; send_password_email | Submission URL for the Forgot Password form `/login.php?action=send_password_email` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:611]:
```suggestion
| &nbsp; &nbsp; &#x21B3; send_password_email | Submission URL for the Forgot Password form. For example, `/login.php?action=send_password_email`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 29]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; save_new_password | Submission URL for saving a new password `/login.php?action=save_new_password` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:612]:
```suggestion
| &nbsp; &nbsp; &#x21B3; save_new_password | Submission URL for saving a new password. For example, `/login.php?action=save_new_password`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 30]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; logout | URL for customer to log out of their account `/login.php?action=logout` |
+| &#x21B3; product | Collection of product-related URLs: |
+| &nbsp; &nbsp; &#x21B3; post_review | URL to submit a Write Review form `/postreview.php`  |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:616]:
```suggestion
| &nbsp; &nbsp; &#x21B3; post_review | URL to submit a Write Review form `/postreview.php`.  |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 31]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; cart | URL to the cart page `/cart.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:618]:
```suggestion
| &#x21B3; cart | URL to the cart page `/cart.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 32]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; checkout  | Collection of checkout related URLs: |
+| &nbsp; &nbsp; &#x21B3; single_address  | URL for the customer to check out with a single shipping address `/checkout.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:620]:
```suggestion
| &nbsp; &nbsp; &#x21B3; single_address  | URL for the customer to check out with a single shipping address `/checkout.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 33]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &#x21B3; multiple_address  | URL for the customer to check out with multiple shipping addresses `/checkout.php?action=multiple` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:621]:
```suggestion
| &nbsp; &nbsp; &#x21B3; multiple_address  | URL for the customer to check out with multiple shipping addresses `/checkout.php?action=multiple`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 34]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; rss | Collection of RSS-related URLs: |
+| &nbsp; &nbsp; &#x21B3; products | Collection of product-feed–related URLs: |
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; new| RSS feed of new products `/rss.php?type=rss` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:624]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; new| RSS feed of new products `/rss.php?type=rss`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 35]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; new_atom | RSS feed of new products, in Atom format `/rss.php?type=atom` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:625]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; new_atom | RSS feed of new products, in Atom format `/rss.php?type=atom`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 36]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; popular | RSS feed of popular products `/rss.php?action=popularproducts&type=rss` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:627]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; popular | RSS feed of popular products `/rss.php?action=popularproducts&type=rss`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 37]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; popular_atom | RSS feed of popular products, in Atom format `/rss.php?action=popularproducts&type=atom` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:629]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; popular_atom | RSS feed of popular products, in Atom format `/rss.php?action=popularproducts&type=atom`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 38]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; featured | RSS feed of featured products `/rss.php?action=featuredproducts&type=rss';` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:630]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; featured | RSS feed of featured products `/rss.php?action=featuredproducts&type=rss';`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 39]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; featured_atom | RSS feed of featured products, in Atom format `/rss.php?action=featuredproducts&type=atom` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:632]:
```suggestion
| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; featured_atom | RSS feed of featured products, in Atom format `/rss.php?action=featuredproducts&type=atom`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 40]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; search | RSS feed of products matching the active search query. |
+| &nbsp; &nbsp; &nbsp; &nbsp; &#x21B3; search_atom | RSS feed of products matching the active search query, in Atom format. |
+| &nbsp; &nbsp; &#x21B3; blog | RSS feed of recent blog posts `/rss.php?action=newblogs&type=rss` |
+| &nbsp; &nbsp; &#x21B3; blog_atom | RSS feed of recent blog posts, in Atom format `/rss.php?action=newblogs&type=atom` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:636]:
```suggestion
| &nbsp; &nbsp; &#x21B3; blog_atom | RSS feed of recent blog posts, in Atom format `/rss.php?action=newblogs&type=atom`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 41]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; contact_us_submit | URL to submit the Contact Us form `/pages.php?action=sendContactForm`|
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:637]:
```suggestion
| &#x21B3; contact_us_submit | URL to submit the Contact Us form `/pages.php?action=sendContactForm`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 42]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; search | URL to submit a search request `/search.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:638]:
```suggestion
| &#x21B3; search | URL to submit a search request `/search.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 43]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; compare | A string containing the URL to the products comparison page `/compare` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:640]:
```suggestion
| &#x21B3; compare | A string containing the URL to the products comparison page `/compare`.  |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 44]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; sitemap | URL to the sitemap `/sitemap.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:650]:
```suggestion
| &#x21B3; sitemap | URL to the sitemap `/sitemap.php`. |

```

File: docs/stencil-docs/reference-docs/global-objects-and-properties.md [hunk 45]
```diff
@@ -622,3 +563,76 @@ To access the global`{{products.top_sellers}}` object on your page, you must fir
...
+| &#x21B3; subscribe | Collection of subscription-related URLs. |
+| &nbsp; &nbsp; &#x21B3; action | URL to which the newsletter subscription will be submitted `/subscribe.php` |
```
Comment (bc-andreadao) [inline docs/stencil-docs/reference-docs/global-objects-and-properties.md:652]:
```suggestion
| &nbsp; &nbsp; &#x21B3; action | URL to which the newsletter subscription will be submitted `/subscribe.php`. |

```

**Review by bc-andreadao — COMMENTED** [review]
Some of the tables on this page (and in our docs) have periods at the end of sentences. I added periods as suggestions, but you might have to add periods for the other tables on this page

**Review by bc-tgomez — APPROVED** [review]
Looks great! Thanks for putting in the work on this one!

**Pull request merged**